components/SearchSandbox/containers/Editor.js

```diff
@@ -486,27 +491,53 @@ export default class Editor extends Component {
 			sortBy: 'asc',
 			paginationAt: 'bottom',
 			...resultComponentProps,
+			renderSuggestions: ({
```

Comment:
@jyash97 Can you explain what this is? `renderSuggestion` is a prop on DataSearch. Refer docs here: https://opensource.appbase.io/reactive-manual/search-components/datasearch.html